linetools/isgm/abssystem.py

```diff
@@ -208,12 +217,17 @@ def add_component(self, abscomp, toler=0.2*u.arcsec):
         comp : AbsComponent
         toler : Angle, optional
           Tolerance on matching coordinates
+        chk_sep : bool, optional
+          Perform coordinate check (expensive)
         """
         # Coordinates
-        test = bool(self.coord.separation(abscomp.coord) < toler)
+        if chk_sep:
+            test = bool(self.coord.separation(abscomp.coord) < toler)
+        else:
+            test = True
         # Now redshift/velocity
-        zlim_comp = (1+abscomp.zcomp)*abscomp.vlim/const.c.to('km/s')
-        zlim_sys = (1+self.zabs)*self.vlim/const.c.to('km/s')
+        zlim_comp = (1+abscomp.zcomp)*abscomp.vlim/constckms
+        zlim_sys = (1+self.zabs)*self.vlim/constckms
```

Comment:
PEP8 appears not to require them.  But the conditionals on the next line do. 
Edited.